templates/haproxy.cfg.j2

```diff
@@ -98,27 +97,27 @@ backend machine-config-server
 frontend ingress-http
     bind *:80
     default_backend ingress-http
-    mode tcp
-    option tcplog
+    mode http
+    option httplog
 
 backend ingress-http
-    balance source
-    mode tcp
+    balance roundrobin
+    mode http
 {% for w in workers %}
-    server {{ w.name | lower }}-http-router{{ loop.index0 }} {{ w.ipaddr }}:80 check
+    server {{ w.name | lower }} {{ w.ipaddr }}:80 check
```

Comment:
Keep the name `-http-router`. It causes confusion otherwise (you'd be surprised).

Reply:
I leave it to you. What kind of confusion?

Reply:
So when people deploy 3 workers, they'll see one of them "red" in the loadbalancer status page. This is because the router only deploys on two workers by default.

Having them labelled as "router" saves me from getting "One of my workers is down" messages (it happens a lot actually :) )

Reply:
Got it. Thanks.